src/operations/create_collection.ts

```diff
@@ -60,6 +60,10 @@ export interface CreateCollectionOptions extends CommandOperationOptions {
   pipeline?: Document[];
   /** A primary key factory function for generation of custom _id keys. */
   pkFactory?: PkFactory;
+  /** A document specifying configuration options for timeseries collections. */
+  timeseries?: Document;
```

Comment:
I think it would nice to provide some hinting here, we can have:
```typescript
interface TimeSeriesOptions extends Document {
  timeField: string;
  metaField: string;
  granularity: string;
}
```

where `extends Document` will permit future options without drivers changes

Reply:
Did you see those specified somewhere? I'm wondering if some of them should be optional.